File: Python/1038.py
num = input().split()
cod = int(num[0])
quant = int(num[1])

if cod == 1:
    print(f'Total: R$ {quant*4:.2f}')
elif cod == 2:
    print(f'Total: R$ {quant*4.5:.2f}')
elif cod == 3:
    print(f'Total: R$ {quant*5:.2f}')
elif cod == 4:
    print(f'Total: R$ {quant*2:.2f}')
elif cod == 5:
    print(f'Total: R$ {quant*1.5:.2f}')

File: C/1038.c
#include <stdio.h>
 
int main() {
    
    int cod, qnt;
    float total;
    scanf("%d%d", &cod, &qnt);
    
    if(cod == 1){
        total = qnt * 4.00;
    }else if(cod == 2){
        total = qnt * 4.50;
    }else if(cod == 3){
        total = qnt * 5.00;
    }else if(cod == 4){
        total = qnt * 2.00;
    }else{
        total = qnt * 1.50;
    }
    
    printf("Total: R$ %.2f\n", total);
}
